tasks/init.js

```diff
@@ -123,23 +124,29 @@ yargs.getInstance('init')
     'The following steps are taken. Some steps are gated by respective a flag. Default options may be globally ' +
     'defined or reset using the --defaults option.',
     '',
-    '* project directory     exists, else create    --subdir',
-    '* /' + padded(20)(streams.APP            ) + ' exists, else create',
-    '* /' + padded(20)(streams.APP + '/*.html') + ' exists, else create',
-    '* /' + padded(20)(streams.APP + '/*.scss') + ' exists, else create',
-    '* angularity.json       exists, else create',
-    '* package.json          exists, else create    --npm',
-    '* bower.json            exists, else create    --bower',
-    '* karma.conf.js         exists, else create    --karma',
-    '* .jshintrc             exists, else create    --jshint',
-    '* .gitignore            exists, else create    --gitignore',
-    '* initialise and launch an IDE                 --ide',
+    '* project directory    exists, else create    --subdir',
+    '* /' + padded(20)(streams.APP            ) + 'exists, else create',
+    '* /' + padded(20)(streams.APP + '/*.html') + 'exists, else create',
+    '* /' + padded(20)(streams.APP + '/*.scss') + 'exists, else create',
+    '* angularity.json      exists, else create',
+    '* package.json         exists, else create    --npm',
+    '* bower.json           exists, else create    --bower',
+    '* karma.conf.js        exists, else create    --karma',
+    '* .jshintrc            exists, else create    --jshint',
+    '* .gitignore           exists, else create    --gitignore',
+    '* run IDE task                                --ide',
     '',
-    'Notes:',
+    'By default, a subdirectory is created for the project',
     '',
-    '* No properties are set in existing files, delete existing files in order to change properties.',
-    '* Both the npm and bower packages are initially set private which you will need to clear in order to publish.',
-    '* Any given IDE is initialised per its task defaults. Use the task separately to review these options.'
+    'Where run on an exising project existing files will not be altered, delete existing files in order to change ' +
+    'properties.',
+    '',
+    'Both the npm and bower packages are initially set private which you will need to clear in order to publish.',
+    '',
+    'Available IDE tasks include:',
+    IDE_LIST.join(', '),
+    '',
+    'Any given IDE is initialised per its task defaults. Use the task separately to review these options.'
   ].join('\n')))
```

Comment:
- This is beginning to look like we should read the description in from a file - there's quite a lot of string concatentation going on here.
- Do either regex replacements or templating for the dynamic bits
- Question: Do we really need those dynamic bits, or would it be OK to simply hard code "app" here?